docs/move/move-on-aptos/objects/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Building with Objects"
+---
+
+import CodeBlock from '@theme/CodeBlock';
+
+# What are objects?
+
+The Object model allows Move to represent a set of resources stored at an
+address with resource control and ownership management.
```

Comment:
How about "The basic Move language controls access to resources using the store ability and accounts.  To better match some common programming styles, the Object model provides a way to associate a collection of resources with a single address, with centralized resource control and ownership management.  An Object is a container for resources that are stored within a single address which can be managed and accessed as a group for more efficiency.  The contract module creating an Object can define custom policies for changes and transfers of those resources as single entity."

Assuming that makes sense outside of my own head.

I would also link to `/standards/aptos-object.md` earlier, since everything you've written is more "technical" than the first page of that doc.  it's fairly readable.

Reply:
It's linked down below.

I think the whole point here is... to make a simpler version since, I'm going to be honest, it was a lot for even me to read.

Let me see if I can submit another one to clean this up, as I think the owners of the Move stuff should own these kinds of docs.